node/nodeHelper_test.go

```diff
@@ -0,0 +1,35 @@
+package node_test
+
+import (
+	"testing"
+
+	"github.com/multiversx/mx-chain-go/config"
+	"github.com/multiversx/mx-chain-go/errors"
+	"github.com/multiversx/mx-chain-go/node"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateNode(t *testing.T) {
+	t.Parallel()
+
+	nodeHandler, err := node.CreateNode(
```

Comment:
Perhaps add two t.Run:
- one for which it should work (either with regular/sovereign factory)
- and this use case with nod node handler